src/languages/en.ts

```diff
@@ -430,7 +430,7 @@ export default {
         chatWithAccountManager: 'Chat with your account manager here',
         sayHello: 'Say hello!',
         welcomeToRoom: ({roomName}: WelcomeToRoomParams) => `Welcome to ${roomName}!`,
-        usePlusButton: '\n\nYou can also use the + button below to request money or assign a task!',
+        usePlusButton: '\n\nYou can also use the + button below to send or request money or assign a task!',
```

Comment:
NAB. This reads a little funny having "or" twice. Perhaps:
```
You can also use the + button to send money, request money, or assign a task!
```

Reply:
😄 Yeah, agree. Thanks for the suggestion.